lib/MIME/Base64.pm

```diff
@@ -67,14 +67,18 @@ argument is the byte string to encode.  The second argument is the
 line-ending sequence to use.  It is optional and defaults to "\n".  The
 returned encoded string is broken into lines of no more than 76
 characters each and it will end with $eol unless it is empty.  Pass an
-empty string as second argument if you do not want the encoded string
-to be broken into lines.
+empty string as the third argument if you do not want the encoded string
```

Comment:
This doesn't look correct. The empty string is passed as the second argument.

Reply:
My mistake.